### PR TITLE
KOGITO-5163 - Dynamic testing of metrics with artifactId and version taken from GAV

### PR DESCRIPTION
**Pull request by r00ta**
JIRA: https://issues.redhat.com/browse/KOGITO-5163
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: dmn-drools-quarkus-metrics/pom.xml [hunk 1]
```diff
@@ -76,6 +76,15 @@
   </dependencies>
   <build>
     <finalName>${project.artifactId}</finalName>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/pom.xml:81]: Can you move `project.properties` in `test/main/resources`?

File: dmn-drools-springboot-metrics/pom.xml [hunk 2]
```diff
@@ -77,6 +77,15 @@
 
   <build>
     <finalName>${project.artifactId}</finalName>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
```
Comment (danielezonca) [inline dmn-drools-springboot-metrics/pom.xml:82]: Same here

File: dmn-drools-quarkus-metrics/pom.xml (hunk 1 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/pom.xml:81]: moved

File: dmn-drools-springboot-metrics/pom.xml (hunk 2 above)
Comment (r00ta) [reply dmn-drools-springboot-metrics/pom.xml:82]: moved

**Review by danielezonca — APPROVED** [review]

**Pull request merged**